pkg/builder/builder.go

```diff
@@ -804,6 +805,16 @@ func BuildStsSpec(nm types.NamespacedName, vdb *vapi.VerticaDB, sc *vapi.Subclus
 				{
 					ObjectMeta: metav1.ObjectMeta{
 						Name: vapi.LocalDataPVC,
+						// Set the ownerReference so that we get auto-deletion
+						OwnerReferences: []metav1.OwnerReference{
+							{
+								APIVersion: vapi.GroupVersion.String(),
+								Kind:       vapi.VerticaDBKind,
+								Name:       vdb.Name,
+								UID:        vdb.UID,
+								Controller: &isControllerRef,
```

Comment:
What do they mean by `Controller`? `verticadb controller`?

Reply:
Yeah, the operator is a controller so we set that to true. We do the same thing for other objects created by the operator.